<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Language (Requires rerun)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="faRadioButton">
        <property name="text">
         <string>Persian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="enRadioButton">
        <property name="text">
         <string>English</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Click on the system tray icon</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="stopPlayRadioButton">
        <property name="text">
         <string>Stop / Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showPlayerWinRadioButton">
        <property name="text">
         <string>Show player window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Player window</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="exitToTryCheckBox">
        <property name="text">
         <string>Exit to try</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="minimizeToTryCheckBox">
        <property name="text">
         <string>Minimize to try</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
